source/common/buffer/watermark_buffer.cc

```diff
@@ -111,13 +111,12 @@ size_t WatermarkBuffer::addFragments(absl::Span<const absl::string_view> fragmen
   return total_size_to_write;
 }
 
-void WatermarkBuffer::setWatermarks(uint32_t high_watermark) {
-  uint32_t overflow_watermark_multiplier =
-      Runtime::getInteger("envoy.buffer.overflow_multiplier", 0);
+void WatermarkBuffer::setWatermarks(uint32_t high_watermark,
```

Comment:
we also need to change this in `TrackedWatermarkBuffer` since it overrides this method but now the signature is changed